### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by tqrahman**
**What is is PR for?**
- [ ] Code update
- [X] Documentation update
- [ ] Both

**What does this PR do?**  
Updates the readme for the repo.

**Is this related to an open issue?**  
#361

**Testing Methodology**  
Nothing to test. Just read

**Additional context**  
Add any other technical detail or considerations here.

**Checklist**
Before you submit this pull request, please make sure you have done the following:

_General_  
- [X] Contribution Guidelines: Have you read the contribution guidelines?  
- [X] Code of Conduct: Have you read the code of conduct?
- [X] Documentation: If applicable, have you added or updated all relevant documentation?

_For Code Updates_  
- [ ] Hardware Validation: If relevant, have you validated your changes on actual supported Duck hardware?
- [ ] Unit Tests: Have you run the unit tests on the device?  
- [ ] Network Testing: If applicable, have you tested your changes on a Duck network?  

Tested Targets (Please check all that apply)  
- [ ] All
- [ ] Heltec LoRa v3 
- [ ] Heltec LoRa v2
- [ ] Heltec CubeCell Series
- [ ] TTGO T-Beam (SX1262)
- [ ] Others

### CODE REVIEW

**Comment by amirna2** [comment]
@tqrahman This looks good to me. We should wait before merging :
1. The CDP Projects are not yet created
2. We need to coordinate these PR as part of a release. We can discuss how we do this and I can share some ways can be done in terms of branching strategies

I am glad to see the new PR template is working nicely.

**Comment by amirna2** [comment]
This PR depends on #366

File: README.md [hunk 1]
```diff
@@ -2,38 +2,31 @@
 
 [![License](https://img.shields.io/badge/License-Apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) [![Slack](https://img.shields.io/badge/Join-Slack-blue)](https://www.project-owl.com/slack) [![Wiki](https://img.shields.io/badge/Read-Wiki-blue)](https://github.com/Call-for-Code/ClusterDuck-Protocol/wiki) [![Build Status](https://travis-ci.com/Call-for-Code/ClusterDuck-Protocol.svg?branch=master)](https://app.travis-ci.com/github/Call-for-Code/ClusterDuck-Protocol)
```
Comment (krook) [inline README.md:3]: Need to update to Discord and remove Slack

File: README.md [hunk 2]
```diff
@@ -131,6 +122,6 @@ v4.0.0
 
 See `library.json` (PlatformIO) or `library.properties` (Arduino).
 
-[Project OWL]: <https://www.project-owl.com/>
-[ClusterDuck Protocol]: <https://github.com/Call-for-Code/ClusterDuck-Protocol/wiki>
-[Slack Workspace]: <https://www.project-owl.com/slack>
+[OWL Integrations]: <https://www.owlintegrations.com/>
+[ClusterDuck Protocol]: <https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/wiki>
+[Slack Workspace]: <https://www.project-owl.com/slack>
```
Comment (krook) [inline README.md:127]: Need to update to Discord and remove Slack

File: README.md [hunk 3]
```diff
@@ -2,38 +2,31 @@
...
 
 ![overview](https://www.project-owl.com/assets/wiki/cdp-explain-gif.gif)
 
 ## Installation
 
-Check out the [Wiki](https://github.com/Call-for-Code/ClusterDuck-Protocol/wiki) to learn how to build your own ClusterDuck Protocol Network. And go to the [ClusterDuck Protocol](https://clusterduckprotocol.org/) website for more information and to learn about projects built upon this codebase. You can reach out directly on our [Slack Workspace] for any questions and work with the community.
+Check out the [Wiki](https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/wiki) to learn how to build your own ClusterDuck Network. Feel free to visit the  [ClusterDuck Protocol](https://clusterduckprotocol.org/) website for more information on projects that the community is building. You can reach out directly on our [Slack Workspace] for any questions and/or to work with the community.
```
Comment (krook) [inline README.md:19]: Need to update to Discord and remove Slack

File: README.md [hunk 4]
```diff
@@ -2,38 +2,31 @@
...
+The network is made up of multiple nodes called **Ducks**. There are 3 basic types of Ducks in a ClusterDuck Network: DuckLink, MamaDuck, and PapaDuck. **DuckLinks** serve as edge nodes that only transmit data. **MamaDucks** inherit the same functionality of the DuckLinks, but can also receive messages. This allows the MamaDucks to relay messages from DuckLinks and/or other MamaDucks along the network (towards the gateway). **PapaDucks** are the endpoint of the network where all of the data is collected and can be stored or pushed up into the cloud. **(free tier cloud DMS platform in BETA! Reach out to us on Slack for more info)**
```
Comment (krook) [inline README.md:13]: Need to update to Discord and remove Slack

**Review by krook — CHANGES_REQUESTED** [review]
Need to update to Discord and remove Slack

**Comment by amirna2** [comment]
Also this probably means https://clusterduckprotocol.org/ will need to be updated to reflect the move to Discord

File: README.md (hunk 1 above)
Comment (tqrahman) [reply README.md]: Changed to Discord and updated the badge!

File: README.md (hunk 2 above)
Comment (tqrahman) [reply README.md]: updated!

File: README.md (hunk 3 above)
Comment (tqrahman) [reply README.md:19]: updated!

File: README.md (hunk 4 above)
Comment (tqrahman) [reply README.md]: updated!

**Comment by TimoWielink** [comment]
> Also this probably means https://clusterduckprotocol.org/ will need to be updated to reflect the move to Discord

Done: https://clusterduckprotocol.org/

**Comment by amirna2** [comment]
@TimoWielink The Logo is not displayed correctly in  `readme` of the `pre-release` branch

image

**Comment by amirna2** [comment]
@tqrahman Are all the changes in for the readme? If so, we can merge this to the pre-release branch.

**Pull request merged**